File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/PersonDetail.php
<?php

namespace Api\Client\Soap\Core;

class PersonDetail
{

    /**
     * @var dateTime $dob
     * @access public
     */
    public $dob = null;

    /**
     * @var string $email
     * @access public
     */
    public $email = null;

    /**
     * @var string $firstName
     * @access public
     */
    public $firstName = null;

    /**
     * @var Gender $gender
     * @access public
     */
    public $gender = null;

    /**
     * @var string $lastName
     * @access public
     */
    public $lastName = null;

    /**
     * @var string $middleName
     * @access public
     */
    public $middleName = null;

    /**
     * @var string $mobile
     * @access public
     */
    public $mobile = null;

    /**
     * @var string $nic
     * @access public
     */
    public $nic = null;

    /**
     * @access public
     */
    public function __construct()
    {
    
    }

}

File: expresslk-services/ExpressLK-core/src/main/java/lk/express/PersonDetail.java
package lk.express;

import java.util.Date;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

import lk.express.bean.Gender;

@XmlRootElement
@XmlType(name = "PersonDetail", namespace = "http://express.lk")
public class PersonDetail {

	private String firstName;
	private String middleName;
	private String lastName;
	private Gender gender;
	private Date dob;
	private String nic;
	private String mobile;
	private String email;

	@XmlElement(nillable = true)
	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	@XmlElement(nillable = true)
	public String getMiddleName() {
		return middleName;
	}

	public void setMiddleName(String middleName) {
		this.middleName = middleName;
	}

	@XmlElement(nillable = true)
	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	@XmlElement(nillable = true)
	public Gender getGender() {
		return gender;
	}

	public void setGender(Gender gender) {
		this.gender = gender;
	}

	@XmlElement(nillable = true)
	public Date getDob() {
		return dob;
	}

	public void setDob(Date dob) {
		this.dob = dob;
	}

	@XmlElement(nillable = true)
	public String getNic() {
		return nic;
	}

	public void setNic(String nic) {
		this.nic = nic;
	}

	@XmlElement(nillable = true)
	public String getMobile() {
		return mobile;
	}

	public void setMobile(String mobile) {
		this.mobile = mobile;
	}

	@XmlElement(nillable = true)
	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}
}
